api/jobs/params/create-update-job-base-parameters.ts

```diff
@@ -30,10 +30,6 @@ export abstract class CreateUpdateJobBaseParameters<T extends CreateUpdateJobBas
     }
 
     setDueDate(dueDate: Date | null): T {
-        if (dueDate && (dueDate.getTime() < (new Date()).getTime())) {
-            throw new SmartlingException("Job Due Date cannot be in the past.");
-        }
-
```

Comment:
Seems like this makes sense only for updating. But ok, let's remove for creating as well

Reply:
This validation is performed by the endpoint itself, so it is redundant in any case.